DTML.EduBot/Dialogs/BasicDialogModule.cs

```diff
@@ -0,0 +1,17 @@
+﻿namespace DTML.EduBot.Dialogs
+{
+    using Autofac;
+
+    public class BasicDialogModule : Module
+    {
+        protected override void Load(ContainerBuilder builder)
+        {
+            base.Load(builder);
+
+            builder.RegisterType<RootDialog>();
```

Comment:
This is where the code would go for Constructor injection. You would need to replace the dependency injection in web api to use this resolver.